app-crypt/swtpm/swtpm-0.10.0.ebuild

```diff
@@ -67,6 +67,7 @@ src_configure() {
 
 src_install() {
 	default
+	use test && rm -fr "${D}/usr/libexec/installed-tests/swtpm"
```

Comment:
This should be ${ED} but we shouldn't install tests based on USE=test anyway, we should do that based on USE=test-install.